cluster/gce/container-linux/README.md

```diff
@@ -0,0 +1,8 @@
+# Container Linux image
+
+The [Container Linux Operating System](https://coreos.com/why/) is a Linux distribution optimized for running containers securely at scale.
+Container Linux provides [an image](https://coreos.com/os/docs/latest/booting-on-google-compute-engine.html) for Google Cloud Platform (GCP).
```

Comment:
This one should be "CoreOS provides" still.

It could be changed to "CoreOS provides [a container linux image]" as well.